### PR TITLE
Prepare v1.7.0-rc1

### PR DESCRIPTION
**Pull request by jrick**
Updates dcrd to release-v1.7.0-rc1 tag.

Updates dcrwallet to release-v1.7.0-rc1 tag.

Updates dcrctl to release-v1.7.0-rc1 tag.

Updates dcrdex to release-v0.4 branch and dcrdex-assets to v0.4.0-rc1
tag.

Updates dcrlnd to master branch.

Updates politeia to v1.3.1 tag.

### CODE REVIEW

**Comment by jrick** [comment]
People are encouraged to build and test this build but this PR will have to wait for at least another update in dcrinstall before it is ready to merge.

**Comment by chappjc** [comment]
I need to get this in dex 0.4 first: https://github.com/decred/dcrdex/pull/1367 (in a couple house).

File: dcrdex/go.mod [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module decred.org/release/dcrdex
 
-go 1.15
+go 1.16
 
 require (
-	decred.org/dcrdex v0.2.0
-	decred.org/dcrdex-assets v0.2.0
+	decred.org/dcrdex v0.0.0-20211217203335-620a0258b8cd
```
Comment (chappjc) [inline dcrdex/go.mod:6]: `decred.org/dcrdex@v0.4.0-rc1` tagged now

File: dcrdex/go.mod [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+	decred.org/dcrdex-assets v0.4.0-rc1
```
Comment (chappjc) [inline dcrdex/go.mod:7]: You can stick with this for `dcrdex-assets`, or go with [3616e97a72c7ce8](https://github.com/decred/dcrdex-assets/commit/3616e97a72c7ce836b2e51d58dc08f2c4cda211d), which is one commit past rc1 with a minor charts display fix.

**Review by chappjc — CHANGES_REQUESTED** [review]

**Review by alexlyp — APPROVED** [review]

**Review by davecgh — APPROVED** [review]
>windows: dist "dcrinstall-manifests" manifest hash: SHA256:e2e29c49d3ee17e0d84cc8cd9a32190e27f4ef66af8952b79200e6624804b615 (go version go1.17.5 windows/amd64)
> linux: dist "dcrinstall-manifests" manifest hash: SHA256:e2e29c49d3ee17e0d84cc8cd9a32190e27f4ef66af8952b79200e6624804b615 (go version go1.17.5 linux/amd64)

**Comment by jholdstock** [comment]
lgtm

```dist "dcrinstall-manifests" manifest hash: SHA256:e2e29c49d3ee17e0d84cc8cd9a32190e27f4ef66af8952b79200e6624804b615 (go version go1.17.5 linux/amd64)```

**Pull request merged**